FpsCheck.py

```diff
@@ -366,33 +364,48 @@ def find_invalid_well_known(self, check_sets):
-                    if field_sym_difference:
-                        self.error_list.append("The following member(s) of " 
-                        + field + " were not present in both the changelist "
-                        + "and .well-known/first-party-set.json file: " + 
-                        str(sorted(field_sym_difference)))
+                well_known_set = FpsSet(
+                    json_schema.get('ccTLDs'), 
+                    json_schema.get('primary'), 
+                    json_schema.get('associatedSites'), 
+                    json_schema.get('serviceSites'))
+                if well_known_set.primary != curr_fps_set.primary:
+                    field_sym_difference = [well_known_set.primary, curr_fps_set.primary]
+                    self.error_list.append("The following primary was not "
+                    + "present in both the changelist and "
+                    + ".well-known/first-party-set.json file: "
+                    + str(sorted(field_sym_difference)))
+                if well_known_set.associated_sites != curr_fps_set.associated_sites:
+                    field_sym_difference = set(
+                       well_known_set.associated_sites) ^ set(curr_fps_set.associated_sites)
+                    self.error_list.append("The following member(s) of " 
+                    + "associatedSites were not present in both the "
+                    + "changelist and .well-known/first-party-set.json "
+                    + "file: " + str(sorted(field_sym_difference)))
+                if well_known_set.service_sites != curr_fps_set.service_sites:
+                    field_sym_difference = set(
+                       well_known_set.service_sites) ^ set(curr_fps_set.service_sites)
+                    self.error_list.append("The following member(s) of " 
+                    + "serviceSites were not present in both the "
+                    + "changelist and .well-known/first-party-set.json "
+                    + "file: " + str(sorted(field_sym_difference)))
+                if well_known_set.ccTLDs != curr_fps_set.ccTLDs:
+                    field_sym_difference = set(
+                        well_known_set.ccTLDS.keys()) ^ set(
+                            curr_fps_set.ccTLDs.keys())
+                    self.error_list.append("The following aliased ccTLD sites"
+                    + "were not present in both the changelist and " 
+                    + ".well-known/first-party-set.json file: "
+                    + str(sorted(field_sym_difference)))
+                    for aliased_site in curr_fps_set.ccTLDs & well_known_set.ccTLDs:
+                        field_sym_difference = set(
+                        well_known_set.ccTLDS[aliased_site]) ^ set(
+                            curr_fps_set.ccTLDs[aliased_site])
+                        self.error_list.append("The following ccTLD aliases"
+                        + "were not present in both the changelist and " 
```

Comment:
```suggestion
                        + "were not present in both the PR and " 
```

"changelist" isn't a term commonly used on GitHub; we should stick to "PR" instead. Same comment elsewhere.